backend/packages/Upgrade/src/api/controllers/FeatureFlagController.ts

```diff
@@ -500,12 +500,12 @@ export class FeatureFlagsController {
     @Body({ validate: true }) exclusionList: FeatureFlagListValidator,
     @Req() request: AppRequest
   ): Promise<FeatureFlagSegmentExclusion> {
-    return this.featureFlagService.addList(exclusionList, 'exclusion', request.logger);
+    return this.featureFlagService.updateList(exclusionList, 'exclusion', request.logger);
```

Comment:
Should we add the test to verify the ids in the body and URL match, like you have for inclusion lists?

Reply:
I was thinking that could be done in another PR for implementing the Exclude List table, but I can also apply that change now. Thanks!